packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -239,15 +240,15 @@ export class VaTextarea {
           )}
         </Host>
       );
-    } else {      
+    } else {
       return (
         <Host>
           <label htmlFor="textarea">
             {label}
             {required && <span class="required">{i18next.t('required')}</span>}
             {hint && <span class="hint-text">{hint}</span>}
           </label>
-          <span id="error-message" role="alert">
+          <span id="input-error-message" role="alert">
```

Comment:
Will changing this ID affect any tests on vets-website?

Reply:
I don't think so, but I don't know 100% - the textarea web component is only being used by 2 of our apps, secure messaging, and a few of the simple forms; but I don't think any of them are checking this ID since RTL doesn't work with shadow DOM as expected.